packages/web/landing-page/package.json

```diff
@@ -8,7 +8,7 @@
   },
   "dependencies": {
     "@radix-ui/react-tooltip": "0.1.7",
-    "@theguild/components": "1.12.1-alpha-bbdd479.0",
+    "@theguild/components": "4.2.4",
```

Comment:
oh no no, we can't merge this one yet. It's waiting for @B2o5T to finish the migration to lates tools

Reply:
me to Kamil, an hour ago:
<img width="381" alt="image" src="https://user-images.githubusercontent.com/56203151/196467253-ea417007-09b8-4cc8-8bcb-d700950ef99b.png">